### PR TITLE
Feat: modal, rating, toast-grid

### PR DESCRIPTION
**Pull request by yashpatel1797**
Added modal, rating, toast and grid components.
[live link](https://vibrant-jang-276ddd.netlify.app/docs.html)

### CODE REVIEW

**Review by kotesh-arya — APPROVED** [review]
Looks consistent and mergeble 👍🏻

File: components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+.dialog {
+  background-color: var(--color-grey-light-1);
+  position: relative;
+  max-width: 50rem;
+  width: 100%;
+  height: 50vh;
+  overflow: auto;
+  display: flex;
+  padding: 15px 30px;
```
Comment (zabihshaik) [inline components/Modal/modal.css]: Better to use rems in place of px if possible

**Review by zabihshaik — COMMENTED** [review]
Everything Looks Good. 
Just a comment. Please look at it.

File: components/Modal/modal.css (hunk 1 above)
Comment (yashpatel1797) [reply components/Modal/modal.css]: thanks, I forgot to replace I think. will update.

**Pull request merged**